charts/osdfir-infrastructure/README.md

```diff
@@ -0,0 +1,295 @@
+<!--- app-name: OSDFIR Infrastructure -->
+# OSDFIR Infrastructure Helm Chart
+
+OSDFIR Infrastructure helps setup Open Source
+Digital Forensics tools to Kubernetes clusters using Helm. 
+
+Currently, OSDFIR Infrastructure
+supports the deployment of the following tools:
+  * Timesketch; ref https://github.com/google/timesketch
+  * Turbinia; ref https://github.com/google/turbinia
+
+## TL;DR
+
+```console
+helm install osdfir-release oci://us-docker.pkg.dev/osdfir-registry/osdfir-charts/osdfir-infrastructure
+```
+> **Tip**: To quickly get started with a local cluster, see [minikube install docs](https://minikube.sigs.k8s.io/docs/start/).
+
+## Introduction
+
+This chart bootstraps a OSDFIR Infrastructure deployment on a [Kubernetes](https://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+
+- Kubernetes 1.19+
+- Helm 3.2.0+
+- PV provisioner support in the underlying infrastructure
+
+> **Note**: Currently Turbinia only supports processing of GCP Persistent Disks and Local Evidence. See [GKE Installations](#gke-installations) for deploying to GKE.
+
+## Installing the Chart
+
+To install the chart, specify any release name of your choice. For example, if you
+want to install the chart for development, you can choose a release name of `osdfir-dev` then run:
+```console
+helm install my-release oci://us-docker.pkg.dev/osdfir-registry/osdfir-charts/osdfir-infrastructure
+```
+The command deploys OSDFIR Infrastructure on the Kubernetes cluster in the default configuration. The [Parameters](#parameters) section lists the parameters that can be configured 
+during installation or see [Installating for Production](#installing-for-production) 
+for a recommended production installation.
+
+## Installing for Production
+
+Pull the chart locally and review the `values-production.yaml` file for a list of values that will be used for production. 
+```console
+helm pull oci://us-docker.pkg.dev/osdfir-registry/osdfir-charts/osdfir-infrastructure --untar
+```
+
+### GKE Installations
+Create a Turbinia GCP account using the helper script in `tools/create-gcp-sa.sh` prior to installing the chart.
+
+Install the chart providing both the original values and the production values,
```

Comment:
What does `providing both the original values and the production values` mean exactly?

Reply:
I think I see now that it means specifying both of those files.  It also looks like this is talking about Turbinia specifically.  Maybe we could clarify that a bit by saying something like `Install the Turbinia chart with the base `values.yaml` config file and the production config file.`

Also, I think this assumes that you are in a specific directory? Should we mention changing to that dir first?

Reply:
It is, but also because Turbinia requires more steps then Timesketch to run on GCP, but the goal was to still install both when running that first install command. I rephrased this to be a bit more clear